pom.xml

```diff
@@ -1736,10 +1736,6 @@
             <groupId>org.apache.hive</groupId>
             <artifactId>hive-storage-api</artifactId>
           </exclusion>
-          <exclusion>
-            <groupId>io.airlift</groupId>
```

Comment:
This is by ORC 1.4.2?

Reply:
Yes. It's a regression introduced by ORC 1.4.1.
ORC 1.4.0 and 1.4.2, 1.4.3 is hiding this in a planned manner.

Reply:
Thanks for the confirmation